src/Components/App/App.css

```diff
@@ -13,12 +13,15 @@
 
 .home-page {
   width: 100vw;
-  height: 100vh;
+  /* height: 100vh; */
   /* background-color: black; */
 }
 
 .background {
   /* margin-top: 10%; */
   z-index: -1;
   position: absolute;
+  /* width: 100vw;
+  height: 100vh; */
+  /* background-repeat: none; */
```

Comment:
We can probably remove these in the future!

Reply:
Good call! I will remove them now!